test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/Assert.cs

```diff
@@ -3,14 +3,17 @@
 
 using System;
 using System.IO;
-using System.Linq;
-using System.Runtime.Serialization;
 using System.Text;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
 {
     internal class Assert : Xunit.Assert
     {
+        // Matches `{filename}: error {code}: {message} [{project}]
+        // See https://stackoverflow.com/questions/3441452/msbuild-and-ignorestandarderrorwarningformat/5180353#5180353
+        private static readonly Regex ErrorRegex = new Regex(@"^(?'location'.+): error (?'errorcode'[A-Z0-9]+): (?'message'.+) \[(?'project'.+)\]$");
```

Comment:
Huh I thought group names went in angle brackets

Reply:
this is supported too